packages/desktop-client/src/components/table.tsx

```diff
@@ -796,15 +797,33 @@ export function TableHeader({
   );
 }
 
-export function SelectedItemsButton({ name, items, onSelect }) {
-  const selectedItems = useSelectedItems();
+type SelectedItemsButtonNames = 'transactions' | 'schedules';
+
+type SelectedItemsButtonProps<T extends MenuItem = MenuItem> = {
+  name: SelectedItemsButtonNames;
+  items: Array<T | typeof Menu.line>;
+  onSelect: (name: string, items: Array<T | typeof Menu.line>) => void;
+};
+
+export function SelectedItemsButton<T extends MenuItem = MenuItem>({
+  name,
+  items,
+  onSelect,
+}: SelectedItemsButtonProps<T>) {
+  const { t } = useTranslation();
+  const selectedItems = useSelectedItems() as Set<T>;
   const [menuOpen, setMenuOpen] = useState(null);
   const triggerRef = useRef(null);
 
   if (selectedItems.size === 0) {
     return null;
   }
 
+  const label = {
+    transactions: t('{{count}} transactions', { count: selectedItems.size }),
```

Comment:
Example with pluralization.

Using natural language keys requires the base `en` translation catalogue to exist to be able to also use plural forms in English. [Example](https://github.com/actualbudget/actual/pull/3036/files#diff-d384b0e5c9aab0820ffbfa8ab4751e5e161758026efc459dd61eaca720420e7bR6-R7)